tests/color/color.mutations.test.js

```diff
@@ -0,0 +1,118 @@
+const { gql } = require('@apollo/client');
+const { setupApp } = require('../helper-functions');
+const {
+  wrongID,
+  color2,
+  error_not_found,
```

Comment:
I think that it would be better to get this error from separate config file with errors.